packages/fetchai/skills/generic_buyer/tests/test_behaviours.py

```diff
@@ -192,70 +203,79 @@ def setup_class(cls):
             ),
         )
 
-    @staticmethod
-    def _check_start_processing_effects(self_, fipa_dialogue, mock_logger) -> None:
+    def setup(self):
```

Comment:
this was effectively used as a setup, which is what I turned it into

Reply:
Why are any of the below changes required? I think it could have been achieved with fewer changes